### PR TITLE
Add Version Increments section

### LINKED ISSUE #45: Describe version increments
The sequence:
1. `/media-profiles` PUT request.
2. vendor's implementation chooses one Media Profile.
3. Node modifies Flow and update its version.
4. Flow update triggers transport file update in IS-05.
5. IS-05 [requires](https://specs.amwa.tv/is-05/releases/v1.1.1/docs/3.1._Interoperability_-_IS-04.html#version-increments) to update IS-04 Sender's version on change made via IS-05.

Maybe we need to add similar Version Increments section to pay reader's attention to this.

### PR DESCRIPTION
**Pull request by N-Nagorny**
Resolves #45.

### CODE REVIEW

**Review by prince-chrismc — COMMENTED** [review]
I have no objection to this addition, but I feel it's already covered with IS-04/05.

Sink properties --> Rx Cap --> versions changes
media profiles --> ST2110 stream --> version change

Glancing as IS-05... https://specs.amwa.tv/is-05/releases/v1.1.1/docs/3.1._Interoperability_-_IS-04.html#version-increments This looks good

File: docs/1.1. Interoperability.md [hunk 1]
```diff
@@ -46,3 +46,9 @@ Example:
 ## Sender and Receiver IDs
 
 The UUIDs used to advertise Senders and Receivers in the Sink Metadata Processing API must match those used in a corresponding IS-04 implementation.
```
Comment (prince-chrismc) [inline docs/1.1. Interoperability.md:48]:
```suggestion
The UUIDs used to advertise Senders and Receivers in the Sink Metadata Processing API must match those used in a corresponding IS-04 and IS-05 implementation.
```

**Comment by prince-chrismc** [comment]
Typo in the opening comment, it fixes #45 😝

File: docs/1.1. Interoperability.md [hunk 2]
```diff
@@ -46,3 +46,9 @@ Example:
...
+
+## Version Increments
+
+In order to prevent unnecessary polling of the Sink Metadata Processing API, changes to Sinks are signalled via the IS-04 versioning mechanism. When the Sink parameters of a Receiver are modified, or when set of Sinks associated with the Receiver is changed, the `version` attribute of the relevant IS-04 Receiver MUST be incremented.
+
+If `/media-profiles` PUT or DELETE operation applied to a Sender causes change of a Source or a Flow, their versions MUST be incrementer as well. If such change causes subsequent change of Connection API `transport_file`, version of the relevant IS-04 Sender MUST be incremented as IS-05 dictates.
```
Comment (N-Nagorny) [inline docs/1.1. Interoperability.md]:
```suggestion
If `/media-profiles` PUT or DELETE operation applied to a Sender causes change of a Source or a Flow, their versions MUST be incremented as well. If such change causes subsequent change of Connection API `transport_file`, version of the relevant IS-04 Sender MUST be incremented as IS-05 dictates.
```

**Pull request merged**